src/components/ConfirmDialog.tsx

```diff
@@ -1,33 +1,40 @@
 import React, { FC } from 'react';
-import { Dialog, DialogTitle, DialogContent, DialogContentText, DialogActions, Button } from '@material-ui/core';
+import { Dialog, DialogTitle, DialogContent, DialogActions, Button } from '@material-ui/core';
 import { useTranslation } from 'react-i18next';
 import ButtonWithProgress from './ButtonWithProgress';
 import styled from 'styled-components';
 
-const StyledDialogContentText = styled(DialogContentText)`
+const StyledDialogContentText = styled.div`
```

Comment:
var det noe rart med `DialogCOntentText`? ville antatt det er noe a11y-greier elns som kommer med der, siden det er en egen komponent for det(?)

Reply:
åja, fordi det kan være andre ting enn tekst også, sikkert?

Reply:
ja, det stemmer. Vil ha muligheten til å legge inn ikon, linker, bilder osv